cinema/views.py

```diff
@@ -21,22 +22,39 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet
+):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
```

Comment:
The `GenreViewSet` does not specify any permission classes. Consider adding permission classes to control access to this viewset, especially if it should be restricted to authenticated users or specific roles.